docs/06-lua/2-reference/12-CREATURE.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Creature
+---
+
+## Creature module
+
+Module for controlling certain creature behavior.
+
+### Functions
+
+- [lua]`trx.creature.are_allies_hostile`  
+    Returns a boolean value representing if allies are currently hostile towards
+    Lara.
+
+- [lua]`trx.creature.set_allies_hostile(enable)`  
+    Sets whether or not allies are hostile towards Lara. 
+    Examples:
+    - [lua]`trx.creature.set_allies_hostile(true)`  
+      Sets allies to be hostile towards Lara
+    - [lua]`trx.creature.set_allies_hostile(false)`  
+      Sets allies to be friendly towards Lara
```

Comment:
WDYT about making it just a `config.flow.allies_hostile` flag? Should reduce lots of code.
I'm not sure how I feel about "configurable" part of such config options – it's controlled by game logic, not a player setting. Though the convenience of forgoing the setters and getters can't be overstated IMO.
As a bonus, we'd get `/set hostile 0` to make monks civil again for free.

Reply:
Thanks, updated 😌 Yes, I'm hoping we can add to this API for the rest of this ticket.
I like that it's a property now, the functions did feel a bit clunky.